### PR TITLE
updated various broken links

### PR DESCRIPTION
**Pull request by runwuf**
updated various broken links

### CODE REVIEW

File: docs/tools/terraform/get-started.rst [hunk 1]
```diff
@@ -13,7 +13,8 @@ Prepare the dependencies
 
     Make sure that you have either the *Administrator* or *Operator* role when creating the API token. When you create a project, you automatically receive the *Administrator* access.
 
-    For more details, refer to the `Project members and roles page <https://docs.aiven.io/docs/platform/concepts/projects_accounts_access.html#project-members-and-roles>`_.
+    For more details, refer to the :doc:`Project member roles page</docs/platform/reference/project-member-privileges>`.
```
Comment (staceysalamon-aiven) [inline docs/tools/terraform/get-started.rst:16]:
```suggestion
    For more information, refer to the :doc:`project member roles page</docs/platform/reference/project-member-privileges>`.
```

Comment (staceysalamon-aiven) [reply docs/tools/terraform/get-started.rst:16]: Thanks for fixing this link!

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Update on the Terraform page looks good! Thanks!

File: docs/products/clickhouse/howto/connect-with-clickhouse-cli.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ To use the ClickHouse® client across different operating systems, we recommend
 
 .. note::
 
-    There are other installation options available for ClickHouse clients for different operating systems. You can find them in `Install ClickHouse-client and ClickHouse-local <https://clickhouse.com/docs/en/integrations/sql-clients/clickhouse-client-local#install-clickhouse-client-and-clickhouse-local>`_ and `Available Installation Options <https://clickhouse.com/docs/en/install/#available-installation-options>`_ in the official ClickHouse documentation.
+    There are other installation options available for ClickHouse clients for different operating systems. You can find them in `Install ClickHouse-client and ClickHouse-local <https://clickhouse.com/docs/en/operations/utilities/clickhouse-local>`_ and `Available Installation Options <https://clickhouse.com/docs/en/install>`_ in the official ClickHouse documentation.
```
Comment (wojcik-dorota) [inline docs/products/clickhouse/howto/connect-with-clickhouse-cli.rst:13]: Please replace the linked docs' titles as follows:
- "Install ClickHouse-client and ClickHouse-local" => "clickhouse-local"
- "Available Installation Options" => "Install ClickHouse"

File: docs/products/clickhouse/howto/connect-with-go.rst [hunk 3]
```diff
@@ -144,5 +144,5 @@ Replace the placeholders in the code with meaningful information on your service
 
 .. seealso::
 
-    * For instructions on how to configure connection settings, see `Connection settings <https://clickhouse.com/docs/en/integrations/go/clickhouse-go/clickhouse-api/#connection-settings>`_.
+    * For instructions on how to configure connection settings, see `Connection settings <https://clickhouse.com/docs/en/integrations/go#connection-details>`_.
```
Comment (wojcik-dorota) [inline docs/products/clickhouse/howto/connect-with-go.rst:147]: Please replace the linked doc' title as follows:

"Connection settings" => "Connection Details"

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

File: docs/products/clickhouse/howto/connect-with-clickhouse-cli.rst [hunk 4]
```diff
@@ -10,7 +10,7 @@ To use the ClickHouse® client across different operating systems, we recommend
 
 .. note::
 
-    There are other installation options available for ClickHouse clients for different operating systems. You can find them in `Install ClickHouse-client and ClickHouse-local <https://clickhouse.com/docs/en/integrations/sql-clients/clickhouse-client-local#install-clickhouse-client-and-clickhouse-local>`_ and `Available Installation Options <https://clickhouse.com/docs/en/install/#available-installation-options>`_ in the official ClickHouse documentation.
+    There are other installation options available for ClickHouse clients for different operating systems. You can find them in `clickhouse-local <https://clickhouse.com/docs/en/operations/utilities/clickhouse-local>`_ and `install ClickHouse <https://clickhouse.com/docs/en/install>`_ in the official ClickHouse documentation.
```
Comment (wojcik-dorota) [inline docs/products/clickhouse/howto/connect-with-clickhouse-cli.rst:13]: Two more fixes please for the doc titles:
ClickHouse local
and
Install ClickHouse

**Review by wojcik-dorota — APPROVED** [review]
Thank you! The ClickHouse part LGTM.

Plz acquire approval from @harshini-rangaswamy for the Kafka part before merging.

File: docs/products/kafka/kafka-mirrormaker/reference/known-issues.rst [hunk 5]
```diff
@@ -3,7 +3,7 @@ Known issues
 
 MirrorMaker 2 may translate offsets incorrectly if LAG shows negative on the target cluster
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
-The following are known issues that are expected to be resolved in version 3.3.0. See  :doc:`recommended tuning paramters</docs/products/kafka/kafka-mirrormaker/reference/advanced-params>` to minimize hitting this issue.
+The following are known issues that are expected to be resolved in version 3.3.0. See  :doc:`recommended tuning paramters</docs/products/kafka/kafka-mirrormaker/concepts/mirrormaker2-tuning>` to minimize hitting this issue.
```
Comment (harshini-rangaswamy) [inline docs/products/kafka/kafka-mirrormaker/reference/known-issues.rst]:
```suggestion
The following are known issues that are expected to be resolved in version 3.3.0. See  :doc:`recommended tuning paramters</docs/products/kafka/kafka-mirrormaker/concepts/mirrormaker2-tuning>`  to mitigate the impact of these issues.
```

**Review by harshini-rangaswamy — APPROVED** [review]
@runwuf, a minor change to the text. The rest looks good. Thanks!

**Review by harshini-rangaswamy — APPROVED** [review]

**Pull request merged**